dbms/src/Storages/KVStore/Decode/DecodingStorageSchemaSnapshot.cpp

```diff
@@ -17,6 +17,94 @@
 
 namespace DB
 {
+DecodingStorageSchemaSnapshot::DecodingStorageSchemaSnapshot(
+    DM::ColumnDefinesPtr column_defines_,
+    const TiDB::TableInfo & table_info_,
+    const DM::ColumnDefine & original_handle_,
+    Int64 decoding_schema_epoch_,
+    bool has_version_column)
+    : column_defines{std::move(column_defines_)}
+    , pk_is_handle{table_info_.pk_is_handle}
+    , is_common_handle{table_info_.is_common_handle}
+    , decoding_schema_epoch{decoding_schema_epoch_}
+{
+    std::unordered_map<ColumnID, size_t> column_lut;
```

Comment:
```suggestion
    std::unordered_map<ColumnID, size_t> column_lut(table_info_.columns.size());
```

Reply:
Oh, I found it is not useful to do this... https://stackoverflow.com/questions/42373232/why-does-stdunordered-map-have-a-reserve-method